MonoGame.Framework/Input/GamePad.UWP.cs

```diff
@@ -92,6 +92,14 @@ private static GamePadState PlatformGetState(int index, GamePadDeadZone deadZone
                 (state.Buttons.HasFlag(WGI.GamepadButtons.X) ? Buttons.X : 0) |
                 (state.Buttons.HasFlag(WGI.GamepadButtons.Y) ? Buttons.Y : 0) |
                 0;
+
+            // Check triggers
+            const double triggerThreshold = 0.1;
```

Comment:
I'm a little concerned about the trigger threshold here.

XInput defines it as:
```
#define XINPUT_GAMEPAD_TRIGGER_THRESHOLD    30
```
https://msdn.microsoft.com/en-us/library/windows/desktop/ee417001(v=vs.85).aspx

XInput defined the triggers as values between 0 and 255.  So to be accurate to XInput the threshold would be a little higher at `~0.1176` or so.